index.rst

```diff
@@ -6,24 +6,219 @@
 
 .. TODO: Delete the note below before merging new content to the main branch.
 
-.. note::
-
-   **This technote is a work-in-progress.**
-
 Abstract
 ========
 
-This is a technote for the description of results from Individual Hardpoint Breakaway Test. 
+This technote describes the analysis and results derived from individual hard point breakaway tests conducted with the M1M3 surrogate.
+The notebook is located within `notebooks_vandv <https://github.com/lsst-sitcom/notebooks_vandv/blob/tickets/SITCOM-838/notebooks/tel_and_site/subsys_req_ver/m1m3/SITCOM-838_Anaysis.ipynb>`_ GitHub repository.
+With all the steps guarded with reasonable timeouts, so problems are detected if hard point cannot travel to reach low or high limit switches, etc.
+**If this test shows that the hard points do not work properly at the limits, this could be one of the blockers for the installation of M1M3 until it is solved.** 
+
+
+Hardpoint Breakaway Test
```

Comment:
I missed here a discussion associated with the stiffness. I know it is a bit obscure and I have been asking Doug to clarify. Here is a message that I got from him in an e-mail:

> When we do a breakaway test the actual length of the hardpoint does not change. However, the displacement mechanisms is operated to apply the tension/compression on the breakaway mechanism. The displacement of the mechanism has to match the inverse of the displacement of the breakaway mechanism. They have to be equal and opposite to retain the same overall length.
>  
> When the breakaway is locked and the load is changed, up there is actually as small change in the overall length due to elastic deformation of the overall system. This provides the slant in the force displacement curve.
>  
> Although the actuator does not actually change length, the actuator thinks it is changing length due to the operation of its displacement mechanism. It reports that it is changing length. A separate LVDT measures the length change of the breakaway mechanism. This LVDT is not presently working correctly. It is reporting both the breakaway LVDT and the displacement actuator LVDT. When Both LFDT are both working correctly, they should be equal and opposite, except over the elastic deformation range.

Ideally, I would ask to have a reference to a document that explains this with details (ask Doug?) and have one paragraph trying to explain it here. I know it is not super easy, and many ppl have difficulties understanding it.

Reply:
I guess that we can check it from Felipe's SPIE paper [ads](https://ui.adsabs.harvard.edu/abs/2018SPIE10700E..3GD/abstract) (sorry I couldn't read it thoroughly). I will put this on the "To-do list"